cetmix_tower_yaml/wizards/cx_tower_yaml_export_wiz.py

```diff
@@ -30,6 +32,7 @@ class CxTowerYamlExportWiz(models.TransientModel):
         help="Remove empty Many2one, Many2many and One2many"
         " field values from the exported YAML file.",
     )
+    preview_code = fields.Boolean()
```

Comment:
maybe rename to `show_preview_code`

Reply:
In this case "preview" is a verb, so it has the very same meaning)
P.S.  In your case "show_code_preview" because "preview" is a noun here.